google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -2206,6 +2275,163 @@ class KubernetesClusterIterator(resource_iter_class_factory(
     """The Resource iterator implementation for Kubernetes Cluster."""
 
 
+class KubernetesNodeIterator(ResourceIterator):
+    """The Resource iterator implementation for KubernetesCluster"""
+
+    def iter(self):
+        """Resource iterator.
+
+        Yields:
+            Resource: KubernetesCluster created
+        """
+        gcp = self.client
+        try:
+            for data, metadata in gcp.iter_kubernetes_nodes(
+                    project_id=self.resource.parent()['projectId'],
+                    zone=self.resource['zone'],
+                    cluster=self.resource['name']):
+                yield FACTORIES['kubernetes_node'].create_new(
+                    data, metadata=metadata)
+        except ResourceNotSupported as e:
+            # API client doesn't support this resource, ignore.
+            LOGGER.debug(e)
+
+
+class KubernetesPodIterator(ResourceIterator):
+    """The Resource iterator implementation for KubernetesCluster"""
```

Comment:
please update all the docstring in the file

Reply:
Done